tests/test_utils.py

```diff
@@ -298,11 +306,536 @@ def check_multistream_classic(fnames):
+                    plt.close()
+
+
+def check_site_files(obs_dir, subdir):
+    return len(os.listdir(os.path.join(obs_dir, subdir))) > 0
+
+
+def get_fluxnet_lat_lon(obs_dir, subdir):
+
+    # get CSV file from site directory to get lat/lng for site
+    try:
+        site_csv_file = glob.glob(os.path.join(obs_dir, subdir, 'AMF*.csv'))[0]
+    except IndexError:
+        site_csv_file = glob.glob(os.path.join(obs_dir, subdir, 'us*.csv'))[0]
+    with open(site_csv_file) as f:
+        second_line = list(f)[1]
+
+    # parse line from header to get lat/lng
+    str_split = tsplit(second_line,
+                       ('Latitude: ', 'Longitude: ', 'Elevation (masl): '))
+    lat = str_split[1].strip()
+    lng = str_split[2].strip()
+
+    return lat, lng
+
+
+def read_fluxnet_obs(subdir, science_test_data_dir, items):
+
+    filename = '%s.stdfmt.hourly.local.txt' % subdir
+
+    # column names for DataFrame (same as VIC variable names)
+    names = ['YEAR', 'MONTH', 'DAY', 'HOUR', 'PREC', 'AIR_TEMP', 'SWDOWN',
+             'LWDOWN', 'OUT_REL_HUMID', 'PRESSURE', 'WIND', 'OUT_EVAP',
+             'SOIL_TEMP_DEPTH1', 'SOIL_TEMP_DEPTH2', 'SOIL_TEMP_DEPTH3',
+             'SOIL_TEMP_DEPTH4', 'SOIL_TEMP_DEPTH5', 'OUT_SOIL_MOIST1',
+             'OUT_SOIL_MOIST2', 'OUT_SOIL_MOIST3', 'OUT_SOIL_MOIST4',
+             'OUT_SOIL_MOIST5', 'OUT_SOIL_TEMP1', 'OUT_SOIL_TEMP2',
+             'OUT_SOIL_TEMP3', 'OUT_SOIL_TEMP4', 'OUT_SOIL_TEMP5', 'OUT_SWNET',
+             'OUT_LWNET', 'OUT_SENSIBLE', 'OUT_LATENT', 'OUT_GRND_FLUX']
+
+    # read in data with -9999.0000 as NaNs
+    obs_dir = os.path.join(science_test_data_dir, 'datasets',
+                           'ec_flux_towers', 'obs')
+    ecflux_df = pd.read_csv(os.path.join(obs_dir, subdir, filename),
+                            skiprows=0,
+                            delim_whitespace=True,
+                            header=None,
+                            names=names,
+                            na_values=-9999.0000)
+
+    # add datetime index
+    time_cols = ['YEAR', 'MONTH', 'DAY']
+    ecflux_df.index = pd.to_datetime(ecflux_df[time_cols])
+
+    if 'HOUR' in ecflux_df:
+        ecflux_df.index += pd.Series([pd.Timedelta(s, unit='s') for s in
+                                     ecflux_df['HOUR'] * 3600],
+                                     index=ecflux_df.index)
+        time_cols.append('HOUR')
+
+    # remove year, day columns of DataFrame
+    ecflux_df.drop(time_cols, inplace=True, axis=1)
+
+    '''if ('HOUR' not in ecflux_df) and ('SEC' in ecflux_df):
+        # add hour column for groupby
+        ecflux_df['HOUR'] = (ecflux_df['SEC'] * (1/3600)
+                                        ).astype(int)'''
+
+    return ecflux_df
+
+
+def plot_fluxnet_comparison(driver, science_test_data_dir,
+                            compare_data_dict,
+                            result_dir, plot_dir,
+                            plots_to_make):
+
+    ''' makes Ameriflux figures
```

Comment:
Cool, thanks. I went a little overboard with adding blank lines for readability.